frontend/plan/src/components/selector/SectionDisplay.js

```diff
@@ -1,23 +1,63 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
+import Badge from "../Badge";
+
 export default class SectionDisplay extends Component {
     stripTime = (s) => {
         let newS = s.replace(" to ", "-");
         newS = newS.replace("on", "");
         return newS;
-    }
+    };
```

Comment:
I understand that this is how we get the data from the Penn Registrar API, but I was wondering if it would be better if this was handled in the backend instead.